Description: Declare maven-bundle-plugin as an extension to build the bundles
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/geronimo-osgi-registry/pom.xml
+++ b/geronimo-osgi-registry/pom.xml
@@ -65,6 +65,8 @@
             <plugin>
                 <groupId>org.apache.felix</groupId>
                 <artifactId>maven-bundle-plugin</artifactId>
+                <version>debian</version>
+                <extensions>true</extensions>
                 <configuration>
                     <instructions>
                         <Export-Package>org.apache.geronimo.osgi.registry.api;version=1.0</Export-Package>
--- a/geronimo-osgi-locator/pom.xml
+++ b/geronimo-osgi-locator/pom.xml
@@ -46,6 +46,7 @@
             <groupId>org.apache.geronimo.specs</groupId>
             <artifactId>geronimo-osgi-registry</artifactId>
             <version>${version}</version>
+            <type>bundle</type>
             <scope>provided</scope>
         </dependency>
         <dependency>
@@ -67,6 +68,8 @@
             <plugin>
                 <groupId>org.apache.felix</groupId>
                 <artifactId>maven-bundle-plugin</artifactId>
+                <version>debian</version>
+                <extensions>true</extensions>
                 <configuration>
                     <instructions>
                         <Import-Package>*</Import-Package>
